### PR TITLE
Adds support for word time offset

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: speech/cloud-client/pom.xml [hunk 1]
```diff
@@ -34,7 +34,7 @@
   </properties>
 
   <!-- FIXME(lesv) - temp to fix an issue w/ GA Datastore -->
```
Comment (lesv) [inline speech/cloud-client/pom.xml]: You could remove this and the commented out stuff.

File: speech/cloud-client/pom.xml [hunk 2]
```diff
@@ -33,25 +33,12 @@
...
-      </dependency>
-    </dependencies>
-  </dependencyManagement>
- -->
-
   <dependencies>
     <!-- [START dependencies] -->
     <dependency>
       <groupId>com.google.cloud</groupId>
       <artifactId>google-cloud</artifactId>
```
Comment (jabubake) [inline speech/cloud-client/pom.xml]: Why do we include `google-cloud` (all client libs) instead of `google-cloud-speech` ?

Comment (gguuss) [inline speech/cloud-client/pom.xml]: I had previously been using google-cloud-speech but this change was requested before.  I'm happy to specify either dependency.

Comment (jabubake) [inline speech/cloud-client/pom.xml]: @gguuss :my  comment should n't block this PR, but I think we want to include `google-cloud-speech` though. We don't recommend using `google-cloud` (brings in all the client libraries).

**Pull request merged**